Include/internal/pycore_pylifecycle.h

```diff
@@ -98,6 +98,12 @@ PyAPI_FUNC(_PyInitError) _Py_PreInitializeFromCoreConfig(
     const _PyCoreConfig *coreconfig,
     const _PyArgv *args);
 
+PyAPI_FUNC(PyObject*) _PyErr_WriteUnraisable(
```

Comment:
Why it is exposed in the header? Should not it be the `static` function?

Reply:
sys.unraisablehook() calls _PyErr_WriteUnraisable().